plugins/modules/helm.py

```diff
@@ -455,6 +473,10 @@ def main():
     # keep helm_cmd_common for get_release_status in module_exit_json
     helm_cmd = helm_cmd_common
     if release_state == "absent" and release_status is not None:
+
+        if not replace and replace:
```

Comment:
I think one of these variables is probably supposed to be different

Reply:
oh My bad. I will update.